packages/code-du-travail-frontend/src/outils/DureePreavisRetraite/steps/Anciennete.tsx

```diff
@@ -0,0 +1,31 @@
+import { Input } from "@socialgouv/cdtn-ui";
+import React from "react";
+import { Field } from "react-final-form";
+
+import { InlineError } from "../../common/ErrorField";
+import { Question } from "../../common/Question";
+import { isPositiveNumber } from "../../common/validators";
+
+function AncienneteStep() {
+  return (
+    <>
+      <Question as="p" required>
```

Comment:
this would be better to use a `label` tag rather than a `p` so user can click on label to focus the input

Reply:
you could use A TextQuestion Component and precise it's a `type="number"`

Reply:
Good idea, I change it 👍